packages/vscode-extension/src/handlers.ts

```diff
@@ -1958,14 +1960,24 @@ export async function migrateTeamsTabAppHandler(): Promise<Result<null, FxError>
         path.join(tabAppPath, "package.json")
       );
       VsCodeLogInstance.warning(warningMessage);
-      window.showWarningMessage(warningMessage, "OK");
+      VS_CODE_UI.showMessage("warn", warningMessage, false, "OK");
     } else {
       // Update codes to use @microsoft/teams-js v2
       await progressBar.next(StringResources.vsc.migrateTeamsTabApp.updateCodes);
       VsCodeLogInstance.info(StringResources.vsc.migrateTeamsTabApp.updateCodes);
-      result = await migrationHandler.updateCodes();
-      if (result.isErr()) {
-        throw result.error;
+      const failedFiles = await migrationHandler.updateCodes();
+      if (failedFiles.isErr()) {
+        throw failedFiles.error;
+      } else {
+        updateFailedFiles = failedFiles.value;
+        if (failedFiles.value.length > 0) {
+          const warningMessage = util.format(
+            StringResources.vsc.migrateTeamsTabApp.updateCodesError,
+            failedFiles.value.join(", ")
```

Comment:
The message may be too long... Imaging there are >3 failed files, the message will be hard to read.
Consider to output number of failed files and one sample file name in the message, and guide user to output for more details.

Reply:
Good concern. Updated. Also updated the screenshot.